static/app/components/nav/utils.tsx

```diff
@@ -24,7 +24,9 @@ export interface NavItemLayout<Item extends NavSidebarItem | NavSubmenuItem> {
 /** SidebarItem is a top-level NavItem which is always displayed in the app sidebar */
 export interface NavSidebarItem extends NavItem {
   /** The icon to render in the sidebar */
-  icon: JSX.Element;
+  icon: ReactElement;
+  /** if te items to display when this SidebarItem is active */
+  overlay?: LazyExoticComponent<any>;
   /** Optionally, the submenu items to display when this SidebarItem is active */
   submenu?: NavSubmenuItem[] | NavItemLayout<NavSubmenuItem>;
   /**
```

Comment:
small nit: we don't have a lint for this, but our general docblock style is pretty much always multi line

```
/**
 * Comment here
 */
```
and never 

```
/** Comment here */
```

Reply:
Reformatted these in f2279cbcd00